bin.js

```diff
@@ -702,6 +702,15 @@ async function processUpload (args, opts = { private: false, ask: false }) {
     for (let i = 0; i < args._.length; i++) {
       const filename = args._[i]
       const htmlFile = `${path.basename(filename).replace('.html', '')}.html`
+
+      // Check if path exists before uploading
+      try {
+        fs.readFileSync(filename)
```

Comment:
What I would do instead is have uploadData reject if there file does not exist... Also, readFileSync is not the right thing to use here. When I'm on the laptop later I'll write up an example.

Reply:
@jasnell - I'm not very familiar with fs. I changed this check to use existsSync within the uploadData fn. Hopefully this is better.